content/micropython-course/course/01.installation/installation-tools.md

```diff
@@ -7,17 +7,17 @@ title: '2. MicroPython Installation Guide'
 description: 'Learn how to install a code editor needed to program your board with MicroPython.'
 ---
 
-## Goals
+## Software Required
 
 In this chapter we will go over how to install the prerequisite software: the **code editor** and the **MicroPython firmware installer**, that will enable you to dive into the learning activities of this course.
```

Comment:
```suggestion

In this chapter we will go over how to install the prerequisite software: the **MicroPython firmware installer** and the **Arduino Labs for MicroPython (Code Editor)**. The installer will install a MicroPython firmware on your board, and the code editor will allow you to program the board with MicroPython.

## Software Required
```

Looks great @jacobhylen , I am suggesting placing the paragraph _before_ the `## Software Requirements`.